PlayerCore/action creators/AdStartTimeoutActionCreator.swift

```diff
@@ -1,6 +1,7 @@
 //  Copyright 2018, Oath Inc.
 //  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
 
-public func adStartTimeoutReached() -> Action {
-    return AdStartTimeout()
+public func adStartTimeoutReached(isVPAID: Bool) -> Action {
+    guard isVPAID else { return AdStartTimeout() }
+    return VPAIDActions.AdStartTimeout()
```

Comment:
What is the difference between those actions?

Reply:
One is a common action for mp4 videos - it will kill the ad and start showing content.
Another one won't kill the ad. Instead, it will make `finalResult` failed and the `selectFinalResultController` will select a new item to be played.